docs/frameworks/nitrojs.md

```diff
@@ -0,0 +1,209 @@
+      <div id="step1-install-yarn">
+      ```
+      yarn add global genezio
+      ```
+      </div>
+  </TabItem>
+</Tabs>
+
+### 2. Ensure you have an Nitro.js App
+
+If you don't have an Express.js app, you can create one using the following steps:
+
+<details>
+  <summary>**Create a basic Nitro.js App**</summary>
+    
+### 1. Initialize a New Nitro.js Project
+
+Run the following command to create a starter template:
+
+<Tabs groupId="packages">
+  <TabItem className="tab-item" value="npm" label="npm">
+      <div id="init-project-npm">
+      ```
+      npx giget@latest nitro nitro-app --install
+      ```
+      </div>
+  </TabItem>
+  
+  <TabItem className="tab-item" value="pnpm" label="pnpm">
+      <div id="init-project-pnpm">
+      ```
+      pnpm dlx giget@latest nitro nitro-app --install
+      ```
+      </div>
+  </TabItem>
+  
+  <TabItem  className="tab-item" value="yarn" label="yarn">
+      <div id="init-project-yarn">
+      ```
+      yarn dlx giget@latest nitro nitro-app --install
+      ```
+      </div>
+  </TabItem>
+</Tabs>
+
+```bash
+cd nitro-app
+```
+### 2. Create a new route
+
+Next, navigate to the `server` directory, and create a new `users.ts` file.
+
+```typescript title="users.ts"
+export default defineEventHandler((event) => {
+  return [
+    { id: 1, name: "Alice" },
+    { id: 2, name: "Bob" },
+  ];
+});
+```
+### 3. Test the Nitro.js App
+
+Run the following command to start the Nitro.js app:
+<Tabs groupId="packages">
+  <TabItem className="tab-item" value="npm" label="npm">
+      <div id="start-app-npm">
+      ```
+      npm run dev
+      ```
+      </div>
+  </TabItem>
+  
+  <TabItem className="tab-item" value="pnpm" label="pnpm">
+      <div id="start-app-pnpm">
+      ```
+      pnpm dev
+      ```
+      </div>
+  </TabItem>
+  <TabItem  className="tab-item" value="yarn" label="yarn">
+      <div id="start-app-yarn">
+      ```
+      yarn dev
+      ```
+      </div>
+  </TabItem>
+</Tabs>
+Open a web browser and navigate to [http://localhost:8080](http://localhost:8080) to see the app running.
+Navigate to [http://localhost:8080/users](http://localhost:8080/users) to see the newly created route.
+</details>
+
+## Deployment Guide
+
+## 1. Build your app for deployment
+```
+nitro build --preset genezio
+```
+## 2. Personalize your generated `genezio.yaml` file
+By building your nitro app using the Genezio preset, a default `genezio.yaml` file will be Generated, which can be further adjusted for specific cases.
+
+This configuration file specifies the project name, deployment region, and details about the backend.
+- [***Read more about Genezio Configuration File***](/docs/project-structure/genezio-configuration-file/)
+  
+  :::info
+1. You might need to replace the `path` field with the path relative at **genezio.yaml** file.
+2. This example configuration works if **genezio.yaml** is in the same directory as your main application file.
+   :::
+## 3. Test Your App Locally
```

Comment:
Isn't this already said starting with line 98?

Reply:
It is, but we are also doing this in the express doc, and I thought it was something we wanted. IMO if we delete one it should for sure be the one in lines 98, but I also see some value in having both, as a way for someone who is creating his first nitro app from scratch to acknowledge that it is working, before doing anything Genezio related.